react/react2/week1/class-exercise/src/App.js

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
these files got me confused

Reply:
May I ask why  and what ? its nice if you describe it and give some suggestions that how should I start it with. Thank you 😊